_includes/cookie-consent.html

```diff
@@ -0,0 +1,68 @@
+<style>
+  #cookie-notice {
+    padding: 0.5rem 1rem;
+    display: none;
+    text-align: center;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background: #222;
+    color: rgba(255,255,255,0.8);
+  }
+  #cookie-notice a {
+    display: inline-block;
+    cursor: pointer;
+    margin-left: 0.5rem;
+  }
+  @media (max-width: 767px) {
+      #cookie-notice span {
+        display: block;
+        padding-top: 3px;
+        margin-bottom: 1rem;
+      }
+      #cookie-notice a {
+        position: relative;
+        bottom: 4px;
+      }
+  }
+</style>
+<div id="cookie-notice"><span>We would like to use third party cookies and scripts to improve the functionality of this website.</span><a id="cookie-notice-accept" class="btn btn-primary btn-sm">Approve</a><a href="/privacy" class="btn btn-primary btn-sm">More info</a></div>
+<script>
+  function createCookie(name,value,days) {
+    var expires = "";
+    if (days) {
+        var date = new Date();
+        date.setTime(date.getTime() + (days*24*60*60*1000));
+        expires = "; expires=" + date.toUTCString();
+    }
+    document.cookie = name + "=" + value + expires + "; path=/";
+  }
+  function readCookie(name) {
+    var nameEQ = name + "=";
+    var ca = document.cookie.split(';');
+    for(var i=0;i < ca.length;i++) {
+        var c = ca[i];
+        while (c.charAt(0)==' ') c = c.substring(1,c.length);
+        if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+    }
+    return null;
+  }
+  function eraseCookie(name) {
+    createCookie(name,"",-1);
+  }
+
+  if(readCookie('cookie-notice-dismissed')=='true') {
+    // Add/include any analytics code or code using cookie or storing
```

Comment:
This is where we would want to put any code (javascript probably) that needs GDPR compliance.